canonical-web-and-design/vanilla-framework
# Removed redundant MAAS docs example

## Done

Removed MAAS custom docs example, since MAAS is now using the docs layout.

Fixes #3452

## QA

- Visit the [demo]()
- See that there is no link to "MAAS docs / Custom docs layout" (compare to [live Vanilla site](https://vanillaframework.io/docs/examples))
- Try to visit /docs/examples/maas-docs and see that it 404s.
